StoredProcedureEFCore/OutputParameter.cs

```diff
@@ -3,17 +3,31 @@
 
 namespace StoredProcedureEFCore
 {
-  internal class OutputParam<T> : IOutParam<T>
-  {
-    public OutputParam(DbParameter param)
+    internal class OutputParam<T> : IOutParam<T>
     {
-      _dbParam = param;
-    }
+        public OutputParam(DbParameter param)
+        {
+            _dbParam = param;
+        }
+
+        public T Value
+        {
+            get
+            {
+                if (_dbParam.Value is System.DBNull)
```

Comment:
Remove the System. Also, are you sure that this condition works ?

Reply:
I have removed the system and updated the if clause.